communication/marketing-team/blog-guidelines.md

```diff
@@ -0,0 +1,63 @@
+
+## Requested Content
+
+We are looking for: 
+
+* TODO << Request specific types of content
+
+Other types of content, like Kubernetes capabilities and tutorials, are better suited for the [SIG-Docs blogging initiative](https://github.com/kubernetes/community/blob/77aa44ba8a506b2d8a7110ff96f545c36db906f3/sig-docs/blog-subproject/README.md).
+
+## Submit a Post
+
+The quickest way to get involved is to let the team in [#sig-contribex](https://kubernetes.slack.com/archives/C1TU9EB9S) that you have an idea for an article.
+
+For now, our official process is to use [SIG-Doc's system](https://github.com/kubernetes/community/blob/77aa44ba8a506b2d8a7110ff96f545c36db906f3/sig-docs/blog-subproject/README.md).
+
+## Blogger Expectations, Responsibilities, and Info
+
+Anyone is welcome to contribute when they have time. 
+
+If you would like to be listed as a member of the team, here are the expectations:
+
+1. Be prepared to write one blog a quarter and participate in edits to other articles. The time commitment is typically 5-10 hours per quarter depending on the number of blog posts in the review queue.
+2. Bloggers are expected to attend at least one upstream marketing team meeting a month or check-in to remain active. 
+3. Remain non-partial: if you receive a request to write about a project, an individual, or a group of people from your employer, you should ask a non-partisan blogger to write it.
+4. As with all contribution to Kubernetes, adhere to the code of conduct, values, and principles of the project.
+
+## How to Write an Effective Blog: 
+
+Keep the following points in mind as you write in order to speed up the review process:
+
+* Use inclusive language understandable by everyone
+  * Rephrase gendered pronouns (change "he" or "she" to "they" or adjust to remove)
+  * Remember nothing is simple when you're starting out (remove "just," "simply", and "easy")
+  * Define terminology or acronyms (do not assume people know what a term means)
+  * Shy away from jargon and colloquial expressions
+  * Writer clearly and avoid ambiguous sentences
+* Emphasize the things you want readers to remember; tell a story
+  * Make sure to have at least one original quote and one picture
+  * Stay on topic and stick to the facts
+  * Provide evidence and data where applicable, to back up your message
+* Be accountable and honest as an author
+  * Remove anything that lacks adequate evidence
+  * Avoid interjecting personal reactions 
+  * Ensure that the blog post is reviewed by the anyone being mentioned in the piece
+  * As the author, never talk about your employer, sell, promote, or pitch; this is about upstream community endeavours and the individuals and groups that create it
+
+## Further Recommendations 
+
+Here are sites that will make you a better writer: 
```

Comment:
nit - can we make this less pejorative, perhaps "Helpful resources for authoring blogs:"